flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/restart/FailureRateRestartStrategy.java

```diff
@@ -92,7 +92,7 @@ public static FailureRateRestartStrategyFactory createFactory(Configuration conf
 		String failuresIntervalString = configuration.getString(
 				ConfigConstants.RESTART_STRATEGY_FAILURE_RATE_FAILURE_RATE_INTERVAL, Duration.apply(1, TimeUnit.MINUTES).toString()
 		);
-		String timeoutString = configuration.getString(ConfigConstants.AKKA_WATCH_HEARTBEAT_INTERVAL, ConfigConstants.DEFAULT_AKKA_ASK_TIMEOUT);
```

Comment:
@tillrohrmann Is it intended that the default for `AKKA_WATCH_HEARTBEAT_INTERVAL` is inherently tied to `DEFAULT_AKKA_ASK_TIMEOUT`?